README.md

```diff
@@ -13,14 +13,14 @@ All backend code and React components should be well-tested
 
 Houdini is designed and tested to run with the following:
 
-* Ruby 2.6
+* Ruby
```

Comment:
Please undo this change. Ruby 2.6 is the minor version of Ruby that is compatible.